tests/core/shape_primitives.output

```diff
@@ -8,6 +8,9 @@ DEAL:: Torus , SD = 0.896586
 DEAL:: Cone , SD = 1.27456
 DEAL:: Cut Hollow Sphere , SD = 0.315918
 DEAL:: Death Star , SD = 1.27344
+DEAL:: Cylinder , SD = 0.00000
+DEAL:: Cylindrical Tube , SD = -0.280625
+DEAL:: Cylindrical Helix , SD = 0.202481
 DEAL::OK
```

Comment:
Nice, but are you sure of these values? it could be nice to eventually have the unit test print out just a simple level set of the geometry. Maybe in a next PR?

Reply:
They make sense given the shapes that are defined but it would be worth it to check all of them (in a next PR)